lib/chai-files.js

```diff
@@ -0,0 +1,108 @@
+var fs = require('fs');
```

Comment:
i would love access to this chai helper from other places, could I convince you this may want to live as its own module?